### PR TITLE
WW: Issuing marathon

### LINKED ISSUE #3516: Add: Surya TV
**Channel Name:** Surya TV
**Country:** India
**Language:** Malayalam
**Website:** https://sunnetwork.in/Default.aspx?Lang=malayalam
**Stream URL (optional):** xxx
**Notes (optional):** There is a paid subscription app for Sun Network who is the owner of this channel. App name is Sun Next. Surya TV is available there.

### LINKED ISSUE #3544: Add: Cartoon network
**Channel Name:** Cartoon network
**Country:** india
**Language:** Hindi
**Website:** 
**Stream URL (optional):** xxx
**Notes (optional):** xxx

### LINKED ISSUE #3688: Add: Persian Channels
List of requests for Persian channels from other issues:

https://github.com/iptv-org/iptv/discussions/3633
please add these channels to persian language
thanks.
If you find a working link to a channel from the list you can add it directly to the playlist by making a pull request, just read this [guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#add-or-replace-a-stream) before you do it.

### LINKED ISSUE #3721: Add: Channel 5
**Channel Name:** Channel 5
**Country:** Singapore
**Language:** English
**Website:** mewatch.sg
**Stream URL (optional):** https://rest-as.ott.kaltura.com/api_v3/service/assetFile/action/playManifest/partnerId/147/assetId/316288/assetType/media/assetFileId/11719964/contextType/PLAYBACK/isAltUrl/False/a.m3u8
**Notes (optional):** 720p stream, no DRM. Seems that 1080p streams use 3 different types of DRM (Widevine, PlayReady, and FairPlay)

### LINKED ISSUE #3722: Add: Channel 8
**Channel Name:** Channel 8
**Country:** Singapore
**Language:** Chinese
**Website:** mewatch.sg
**Stream URL (optional):** https://rest-as.ott.kaltura.com/api_v3/service/assetFile/action/playManifest/partnerId/147/assetId/260581/assetType/media/assetFileId/11719965/contextType/PLAYBACK/isAltUrl/False/a.m3u8
**Notes (optional):**

### LINKED ISSUE #3723: Add: Suria
**Channel Name:** Suria
**Country:** Singapore
**Language:** Malay
**Website:** mewatch.sg
**Stream URL (optional):** https://rest-as.ott.kaltura.com/api_v3/service/assetFile/action/playManifest/partnerId/147/assetId/237652/assetType/media/assetFileId/11719963/contextType/PLAYBACK/isAltUrl/False/a.m3u8
**Notes (optional):** xxx

### LINKED ISSUE #3726: Add: Canal 13
**Channel Name:** Canal 13 Sinart
**Country:** Costa Rica
**Language:** Spanish
**Website:** https://costaricamedios.cr/envivo
**Stream URL (optional):** https://stream-03.nyc.dailymotion.com/sec(0HE75RGSqqnhnSdEMl12Zb2HJIphEv5GlmKxpQKnXyY)/dm/3/x7vh8g3/d/live-2.m3u8
**Notes (optional):** Another stream URL, perhaps of different image quality https://dai.ly/x7vh8g3

### LINKED ISSUE #3830: Add: itv4(akamai Link)
**Channel Name:** itv4
**Country:** UK
**Language:** English
**Website:** xxx
**Stream URL (optional):** http://sportportal1.akamaized.net/hls/live/702647/itvlive/ITV4PORTAL2/master_Main600.m3u8
**Notes (optional):**  akamai Link.

### LINKED ISSUE #3856: *Broken Link for NHK BS1*
**_Channel Name:_** NHK BS1
**_Broken Link (from playlist):_** http://redlabmcdn.s.llnwi.net/jp04/bs01hd/playlist.m3u8

### LINKED ISSUE #3859: Fix: TOKYO MX1
**_Channel Name:_** TOKYO MX1
**_Broken Link (from playlist):_** http://redlabmcdn.s.llnwi.net/jp04/ryowa8hd/playlist.m3u8
**_Possible Replacement (optional):_** https://neetball.net/live/test3.m3u8

All Ryowa Link said "403 Unauthorized"

### PR DESCRIPTION
**Pull request by Dum4G**
Closes https://github.com/iptv-org/iptv/issues/3830
Closes https://github.com/iptv-org/iptv/issues/3726
Closes https://github.com/iptv-org/iptv/issues/3723
Closes https://github.com/iptv-org/iptv/issues/3722
Closes https://github.com/iptv-org/iptv/issues/3721
Closes https://github.com/iptv-org/iptv/issues/3593
Closes https://github.com/iptv-org/iptv/issues/3516
Closes https://github.com/iptv-org/iptv/issues/3688
Closes https://github.com/iptv-org/iptv/issues/3859
Closes https://github.com/iptv-org/iptv/issues/3862
Closes https://github.com/iptv-org/iptv/issues/3861
Closes https://github.com/iptv-org/iptv/issues/3856
Closes https://github.com/iptv-org/iptv/issues/3863
Closes https://github.com/iptv-org/iptv/issues/3544

### CODE REVIEW

File: channels/uk.m3u [hunk 1]
```diff
@@ -309,12 +309,18 @@ https://live.playstop.me/1816184091/index.m3u8
 https://live.islamchannel.tv/islamtv/islamtv_english/bitrate1.isml/live.m3u8
 #EXTINF:-1 tvg-id="ITV.uk" tvg-name="ITV" tvg-country="UK" tvg-language="" tvg-logo="https://i.imgur.com/SsgOgRF.png" group-title="",ITV
 https://sportportal1.akamaized.net/hls/live/702646/itvlive/ITV1PORTAL2/master.m3u8
+#EXTINF:-1 tvg-id="ITV.uk" tvg-name="ITV" tvg-country="UK" tvg-language="" tvg-logo="https://i.imgur.com/SsgOgRF.png" group-title="",ITV
```
Comment (tilfizyawn) [inline channels/uk.m3u:312]:
```suggestion
#EXTINF:-1 tvg-id="ITV.uk" tvg-name="ITV" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/SsgOgRF.png" group-title="",ITV
```

File: channels/uk.m3u [hunk 2]
```diff
@@ -309,12 +309,18 @@ https://live.playstop.me/1816184091/index.m3u8
...
+#EXTINF:-1 tvg-id="ITV2.uk" tvg-name="ITV2" tvg-country="UK" tvg-language="English" tvg-logo="" group-title="",ITV2
+http://31.220.41.88:8081/live/itv2.stream/chunks.m3u8
 #EXTINF:-1 tvg-id="ITV2.uk" tvg-name="ITV2" tvg-country="UK" tvg-language="English" tvg-logo="" group-title="",ITV2
-http://51.52.156.22:8888/http/005
+http://93.190.139.35:8278/streams/d/itv2_antik/playlist.m3u8
 #EXTINF:-1 tvg-id="ITV3.uk" tvg-name="ITV3" tvg-country="UK" tvg-language="English" tvg-logo="" group-title="",ITV3
-http://51.52.156.22:8888/http/006
+http://31.220.41.88:8081/live/itv3.stream/chunks.m3u8
+#EXTINF:-1 tvg-id="ITV4.uk" tvg-name="ITV4" tvg-country="UK" tvg-language="English" tvg-logo="" group-title="",ITV4
+http://sportportal1.akamaized.net/hls/live/702647/itvlive/ITV4PORTAL2/master.m3u8
```
Comment (tilfizyawn) [inline channels/uk.m3u:321]:
```suggestion
https://sportportal1.akamaized.net/hls/live/702647/itvlive/ITV4PORTAL2/master.m3u8
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by LaneSh4d0w** [comment]
Persiana Classic : http://51.210.199.26/hls/stream.m3u8

**Comment by LaneSh4d0w** [comment]
http://51.210.199.28/hls/stream.m3u8 Indicates 4 HD... sounds like another channel of the 4Turk category.

**Review by Sphinxroot — APPROVED** [review]

**Comment by MH140000** [comment]
http://51.210.199.30/hls/stream.m3u8 sepanj tv
http://51.210.199.1/hls/stream.m3u8 Toheed tv

**Comment by MH140000** [comment]
I would be happy to help if you would like help with Iranian streams.

**Pull request merged**